cmd/encryption-v1.go

```diff
@@ -447,6 +450,18 @@ func (o *ObjectInfo) DecryptedSize() (int64, error) {
 	return size, nil
 }
 
+// EncryptedOffsetByte returns the sequence number of the object after
+// encryption. An encrypted object is always larger than a plain object
+// except for zero size objects.
+func (o *ObjectInfo) EncryptedOffsetByte(s int64) (uint32, int64) {
+	if s == 0 {
+		return 0, 0
+	}
+	seqNumber := s / (64*1024 + 32)
+	offsetByte := seqNumber % (64*1024 + 32)
```

Comment:
s / `seqNumber % (64*1024 + 32)` / `s % (64*1024 + 32)`

`seqNumber % (64*1024 + 32)` is always 0 since `seqNumber = s/(64*1024 + 32)`

Reply:
this doesn't yield correct value lets say with a original 12 byte object is 44 bytes on disk, if i am trying to read from original offset `2` doing `s % (64*1024 + 32)`  == `2 %  (64*1024 + 32)` == 2 , which is an incorrect offset since the original length is '44'  and '44 + 2' is not correct. So it would generally mean that we always have to read from the beginning of the file and skip bytes. 

> seqNumber = s/(64*1024 + 32)

This also this means that we are always reading from the beginning.

Reply:
Oh `offsetByte` is the offset you want to start to **read** from?! Okay than you're completely right about the wrong offset. But for **reading** the `offsetByte` must point to the beginning of the package. For all objects < 64 KB the current code should work but for larger objects where you try to access a position > 64KB I would expect it to fail.

`offsetByte` should point to the first byte of the `seqNumber`-package. So:
`offsetByte = seqNumber * (64*1024 + 32)`. That way `offsetByte` always points to the first byte of the requested package.

Reply:
yes this is what i would expect.